pkg/apis/pipeline/v1alpha1/verificationpolicy_types.go

```diff
@@ -90,6 +90,10 @@ type KeyRef struct {
 	// Data contains the inline public key.
 	// +optional
 	Data string `json:"data,omitempty"`
+	// KMS contains the KMS url of the public key
+	// Supported formats differ based on the KMS system used.
+	// +optional
+	KMS string `json:"kms,omitempty"`
```

Comment:
We should briefly comment what the format the kms url should follow, and also document in [trusted-resources.md](https://github.com/tektoncd/pipeline/blob/main/docs/trusted-resources.md). Are we planning to use sigstore package to do the signing? If so, https://docs.sigstore.dev/cosign/kms_support/ might be a good ref to link to.

Reply:
In addition to the format, I suggest providing a concrete example. "A picture is worth 100 words." :-)

Reply:
> In addition to the format, I suggest providing a concrete example. "A picture is worth 100 words." :-)

a potential example makes sense! I also added a note as you suggested